index.bs

```diff
@@ -522,9 +540,36 @@ apply to the shadow host chain.
 
 <em>This section is non-normative.</em>
 
-The API applies only to {{HTMLVideoElement}} in order to start on a minimal
-viable product that has limited security issues. Later versions of this
-specification may allow PIP-ing arbitrary HTML content.
+The {{HTMLVideoElement}} Picture-in-Picture API has limited security issues.
+
+## Restrictions on arbitrary content ## {#arbitrary-content-restrictions}
+
+The <a>Picture-in-Picture browsing context</a> MUST have top-level navigation
+blocked and the following <a>sandboxing flag set</a> applied:
+- <a>sandboxed navigation browsing context flag</a>
+- <a>sandboxed auxiliary navigation browsing context flag</a>
+- <a>sandboxed forms browsing context flag</a>
+- <a>sandboxed pointer lock browsing context flag</a>
+- <a>sandboxed document.domain browsing context flag</a>
+- <a>sandbox propagates to auxiliary browsing contexts flag</a>
+- <a>sandboxed modals flag</a>
+- <a>sandboxed orientation lock browsing context flag</a>
+- <a>sandboxed presentation browsing context flag</a>
+
+A media element in the Picture-in-Picture window should always be
+<a>allowed to play</a>
+
+If the <a>Picture-in-Picture browsing context</a> is showing
+<a>non-interactive arbitrary content</a> then the user agent MUST not allow
+<a>user interaction</a> with the document.
+
+If the <a>Picture-in-Picture browsing context</a> is showing <a>interactive
+arbitrary content</a> then the user agent MUST allow ANY touch, gesture and
+mouse user input events. The user agent MUST only allow keyboard events from
+the following key tables:
+- <a>control characters</a>
+- <a>modifier keys</a>
+- <a>navigation keys</a>
```

Comment:
Should we allow "whitespace keys" for tabbing and enter?

Reply:
Control characters include tabbing and enter